README.md

```diff
@@ -1,14 +1,891 @@
+                            "PyeeFinSvcsPrvdrComssn": {
+                                "Amt": {
+                                    "Amt": 30.71,
+                                    "Ccy": "USD"
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "SplmtryData": {
+            "Envlp": {
+                "Doc": {
+                    "InitgPty": {
+                        "Glctn": {
+                            "Lat": "-3.1609",
+                            "Long": "38.3588"
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+## Pacs002 Message
+
+**Sample Request Body (ISO20022 Pacs002)**
+
+```json
+{
+    "TxTp": "pacs.002.001.12",
+    "FIToFIPmtSts": {
+        "GrpHdr": {
+            "MsgId": "30bea71c5a054978ad0da7f94b2a40e9789",
+            "CreDtTm": "2021-12-03T15:24:27.000Z"
+        },
+        "TxInfAndSts": {
+            "OrgnlInstrId": "5ab4fc7355de4ef8a75b78b00a681ed2255",
+            "OrgnlEndToEndId": "2c516801007642dfb892944dde1cf845897",
+            "TxSts": "ACCC",
+            "ChrgsInf": [
+                {
+                    "Amt": {
+                        "Amt": 307.14,
+                        "Ccy": "USD"
+                    },
+                    "Agt": {
+                        "FinInstnId": {
+                            "ClrSysMmbId": {
+                                "MmbId": "dfsp001"
+                            }
+                        }
+                    }
+                },
+                {
+                    "Amt": {
+                        "Amt": 153.57,
+                        "Ccy": "USD"
+                    },
+                    "Agt": {
+                        "FinInstnId": {
+                            "ClrSysMmbId": {
+                                "MmbId": "dfsp001"
+                            }
+                        }
+                    }
+                },
+                {
+                    "Amt": {
+                        "Amt": 30.71,
+                        "Ccy": "USD"
+                    },
+                    "Agt": {
+                        "FinInstnId": {
+                            "ClrSysMmbId": {
+                                "MmbId": "dfsp002"
+                            }
+                        }
+                    }
+                }
+            ],
+            "AccptncDtTm": "2021-12-03T15:24:26.000Z",
+            "InstgAgt": {
+                "FinInstnId": {
+                    "ClrSysMmbId": {
+                        "MmbId": "dfsp001"
+                    }
+                }
+            },
+            "InstdAgt": {
+                "FinInstnId": {
+                    "ClrSysMmbId": {
+                        "MmbId": "dfsp002"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+## Pacs008 Message
```

Comment:
pacs.008 should probably be listed before pacs.002.

Reply:
Done.